lib/purescript-cst/src/Language/PureScript/CST/Layout.hs

```diff
@@ -265,7 +265,7 @@ insertLayout src@(SourceToken tokAnn tok) nextPos stack =
         ((_, LytTick) : stk', acc') ->
           (stk', acc') & insertToken src
         _ ->
-          state & insertDefault & pushStack tokPos LytTick
+          state & collapse offsideEndP & insertSep & insertToken src
```

Comment:
You still need to `pushStack`. We should only be replaced the `insertDefault`.

Reply:
Ah... Now I see.